pandas/plotting/_core.py

```diff
@@ -2753,20 +2767,30 @@ def hist(self, by=None, bins=10, **kwds):
         """
         return self(kind='hist', by=by, bins=bins, **kwds)
 
-    def kde(self, **kwds):
+    def kde(self, bw_method=None, ind=None, **kwds):
         """
         Kernel Density Estimate plot
 
         Parameters
         ----------
+        bw_method: str, scalar or callable, optional
+            The method used to calculate the estimator bandwidth.  This can be
+            'scott', 'silverman', a scalar constant or a callable.
+            If None (default), 'scott' is used.
+            See :scipy:class:`stats.gaussian_kde` for more information.
```

Comment:
Have you verified that this link works? I thought it would be

    :func:`scipy.stats.gaussian_kde`.

Reply:
Agreed. Changed it.